===================================================================
RCS file: /cvs/gnome/procman/src/load-graph.c,v
retrieving revision 1.33
retrieving revision 1.33.2.1
diff -u -r1.33 -r1.33.2.1
--- procman/src/load-graph.c	2004/11/12 08:20:07	1.33
+++ procman/src/load-graph.c	2004/12/24 14:33:10	1.33.2.1
@@ -200,10 +200,10 @@
 	glibtop_get_swap (&swap);
 
 	swappercent = (float)swap.used / (float)swap.total;
-	mempercent  = (float)mem.used  / (float)mem.total;
+	mempercent  = (float)mem.user  / (float)mem.total;
 
 	text1 = gnome_vfs_format_file_size_for_display (mem.total);
-	text2 = gnome_vfs_format_file_size_for_display (mem.used);
+	text2 = gnome_vfs_format_file_size_for_display (mem.user);
 	text3 = g_strdup_printf ("  %.1f %%", mempercent * 100.0f);
 	gtk_label_set_text (GTK_LABEL (g->memused_label), text2);
 	gtk_label_set_text (GTK_LABEL (g->memtotal_label), text1);
